frontend/src/components/common/ApiVersionInfo.test.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { ApiVersionInfo } from './ApiVersionInfo';
+import renderer, { act } from 'react-test-renderer';
+
+const defaultVersion = {
+  environment: 'test',
+  version: '11.1.1.1',
+};
+
+const mockGetVersion = jest.fn(async () => {
+  return Promise.resolve({ data: defaultVersion });
+});
+
+jest.mock('hooks/pims-api', () => ({
+  useApiHealth: () => ({
+    getVersion: mockGetVersion,
+  }),
+}));
+
+describe('ApiVersionInfo suite', () => {
+  afterEach(() => {
+    mockGetVersion.mockClear();
+  });
+
+  it('Displays version component', () => {
+    const result = renderer.create(<ApiVersionInfo />).toJSON();
+    expect(result).toMatchSnapshot();
+  });
+
+  it('Displays version information', async () => {
+    await act(async () => {
+      const { findByTestId } = render(<ApiVersionInfo />);
+      const element = await findByTestId('version');
+      expect(element).toContainHTML(`v${defaultVersion.version}`);
+      expect(mockGetVersion).toHaveBeenCalledTimes(4);
```

Comment:
@devinleighsmith This gets called 4 times because there are two tests in this suite.  I thought `mockClear()` would resolve this, but it doesn't appear to work.  If I use `mockReset()` then the whole thing fails to work.
Additionally, it shouldn't be getting called 2 times each test either.  For some reason the `useApi` hook fires twice, even though it uses empty array `[]`.  I'm not sure how to debug.